src/assignment/assignment.service.ts

```diff
@@ -1,5 +1,6 @@
 import { fetchWithLogoutJson } from '@meemoo/admin-core-ui';
 import { type Avo } from '@viaa/avo2-types';
+import { type UserSchema } from '@viaa/avo2-types/types/user';
```

Comment:
Don't use deep imports
import Avo
```suggestion
import { type Avo } from '@viaa/avo2-types';
```

And then use this instead of UserSchema. You should never use types with Schema at the end. That's the internal naming for the typings repo
```
Avo.User.User
```